cmd/yggd/config.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	pb "github.com/redhatinsights/yggdrasil/protocol"
+	"github.com/urfave/cli/v2"
+)
+
+func getConfig(c *cli.Context) *pb.Config {
+	config := &pb.Config{
```

Comment:
If we're going to store configuration state, I think I would rather abstract it inside a generic `main.Config` struct rather than directly into a protobuf message. protobuf isn't going to be the only wire format in use eventually, so this would set us up for supporting multiple RPC protocols.